### PR TITLE
Enhancement/3012 analytics account summaries

### PR DESCRIPTION
**Pull request by eugene-manuilov**
## Summary

Addresses issue #3012

## Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [ ] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

**Comment by felixarntz** [comment]
@eugene-manuilov Please keep in mind to connect PRs to their respective issues in the future (via ZenHub UI) :)

**Comment by eugene-manuilov** [comment]
@felixarntz it doesn't work anymore. It stopped working last week :(

**Comment by felixarntz** [comment]
Hmm weird, it does for me. You don't see the UI at all or how is it not working?

**Comment by eugene-manuilov** [comment]
@felixarntz this is what I see when I try to connect this PR with the issue:

github com_google_site-kit-wp_pull_3081

**Comment by eugene-manuilov** [comment]
Please, ignore the wrong issue number on my screenshot. It behaves the same when I try to connect a PR with a correct issue.

File: includes/Modules/Analytics.php [hunk 1]
```diff
@@ -1291,4 +1266,34 @@ private function register_tag() {
 		}
 	}
 
+	/**
+	 * Finds a property in the properties list.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array  $properties  An array of Analytics properties to search in.
+	 * @param string $property_id Optional. The Analytics property ID. Default the current property ID from the Analytics settings.
```
Comment (tofumatt) [inline includes/Modules/Analytics.php:1275]:
```suggestion
	 * @param string $property_id Optional. The Analytics property ID. Default is the current property ID from the Analytics settings.
```

**Review by tofumatt — COMMENTED** [review]
Looks good; one tiny documentation tweak but it's just a single word so I'll make it then approve 🙂

**Comment by tofumatt** [comment]
@googlebot I consent.

**Review by tofumatt — COMMENTED** [review]
This looks great—so much simpler!

Can you add a QA brief to the issue? After that we can merge it 👍🏻

**Comment by eugene-manuilov** [comment]
@tofumatt QAB is added.

**Review by tofumatt — APPROVED** [review]
Code is a lot less complicated which is great 👍🏻 

I tested this locally and it worked for me. Nice!

**Pull request merged**